_docs/_latest/setup/upgrade.md

```diff
@@ -114,12 +114,13 @@ for example, `deploy-forseti-server-79c4374.yaml`.
 
 ### Change deployment properties
 
-1. Review `deploy-forseti-server.yaml.sample` and `deploy-forseti-client.yaml.sample` for any
-new properties that you need to copy to your previous deployment template. To compare what's changed, use
-the `git diff` command. For example, to see the diff between the latest (HEAD) and the most recent revision, run:
+1. Review [`deploy-forseti-server.yaml.in`](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/deployment-templates/deploy-forseti-server.yaml.in) 
+and [`deploy-forseti-client.yaml.in`](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/deployment-templates/deploy-forseti-client.yaml.in) 
+for any new properties that you need to copy to your previous deployment template. To compare what's changed, use
+the `git diff` command. For example, to see the diff between v2.1.0 and v2.2.0, run:
 
    ```bash
-   $ git diff origin..HEAD~1 -- deploy-forseti-server.yaml.sample
+   $ git diff v2.1.0..v2.2.0 -- deployment-templates/deploy-forseti-server.yaml.in
```

Comment:
Is there any more reasons to have the `.sample` file anymore, if we don't use this here?  The sample file doesn't even look like a real sample.

Reply:
The sample files are outdated at this moment, we should either delete the file or make sure that they are up-to-date.

Reply:
Okay, let's remove the sample file separately.  Thanks!